src/classes/OPP_AutomatedSoftCreditsService.cls

```diff
@@ -0,0 +1,51 @@
+/*
+    Copyright (c) 2018, Salesforce.org
+    All rights reserved.
+    
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+    
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE 
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, 
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER 
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/**
+* @author Salesforce.org
+* @date 2018
+* @group Opportunity
+* @description Handles Automated Soft Credits for Individual and Organizational gifts.
+*/
+
+public class OPP_AutomatedSoftCreditsService {
+
+    /*********************************************************************************************************
+    * @description Determines if the Account is an Organization based on its account classification, 
+    *               e.g., Individual.
+    * @param accountClassification The classification of an Opportunity's associated Account.
+    * @return Boolean
+    **********************************************************************************************************/
+    public static Boolean isOrganizationalAccount(Boolean accountClassification) {
```

Comment:
I know we disagree on this, and I don't like forcing people to a design that isn't there's, but this still makes no sense to me.  we've created an entire service class and testing class, for a single method that take an isIndividualAccount boolean, and returns whether it is false.  I still fail to see the benefit of this service class or method.  It isn't adding any abstraction, since all callers have to know to pass the correct boolean field, and that field has even been documented in your apex methods that use it that it specifically says whether the class is an Individual account or not.  Organizational account is just the opposite of Individual account.  there is no third choice in NPSP.   Would you be open to having Jenny and Radmila taking a look and sharing their thoughts, and we go with the majority preference?

Reply:
@davidhabib -- I'm totally open to having a group discussion about the design. Team discussion is great! Also, please note the code for the Relationships and Affiliations handling will be added to the new service class.

Reply:
ah, glad to hear this class is intended for more than just this one method.  given that, let's go ahead and move forward with what you have.  thanks.

Reply:
Sounds good.